UnityProject/Assets/Scripts/Messages/Client/VariableViewer/RequestInvokeFunction.cs

```diff
@@ -20,14 +21,15 @@ private void ValidateAdmin(NetMessage msg)
 		{
 			if (IsFromAdmin() == false) return;
 
-			global::VariableViewer.RequestInvokeFunction(msg.PageID, SentByPlayer.GameObject, SentByPlayer.UserId);
+			global::VariableViewer.RequestInvokeFunction(msg.PageID,msg.SendToClient , SentByPlayer.GameObject, SentByPlayer.UserId);
 		}
 
-		public static NetMessage Send(ulong _PageID)
+		public static NetMessage Send(ulong _PageID, bool InSendToClient)
```

Comment:
why do you always name your variables "inX"? I don't get the value of prefixing them with "in"

Reply:
I think it's a habit from dependency injection + Clashing with class Fields